dev-python/pygtk/files/pygtk-2.24.0-pango-1.44.patch

```diff
@@ -0,0 +1,39 @@
+From 877164b6b70780468a31d8211f29421b6f34b0c8 Mon Sep 17 00:00:00 2001
```

Comment:
I suggest to refer to commit 4aaa48eb80c6802aec6d03e5695d2a0ff20e0fc2 in pygtk upstream repository.